nautobot_chatops/dispatchers/slack.py

```diff
@@ -23,7 +23,7 @@
 BACKEND_ACTION_BLOCKS = backend_action_sum.labels("slack", "send_blocks")
 BACKEND_ACTION_SNIPPET = backend_action_sum.labels("slack", "send_snippet")
 
-SLACK_MAX_MESSAGE_LENGTH = 40000
```

Comment:
Was SLACK_MAX_MESSAGE_LENGTH really not used anywhere else? That surprises me.

Reply:
Yeah, I basically changed the name of the constant variable to reflect that more proper.